.eslintignore

```diff
@@ -9,3 +9,4 @@ picogl/
 .docusaurus
 
 test/apps/
+bundle.ts
```

Comment:
You added bundle.ts files in this PR, why are we also gitignoring them?

Reply:
This is eslintignore